main.go

```
@@ -0,0 +1,133 @@
+
+func newModel() model {
+	// Create sample items
+	items := []list.Item{
+		item{title: "Item 1", description: "First item"},
+		item{title: "Item 2", description: "Second item"},
+		item{title: "Item 3", description: "Third item"},
+	}
+
+	// Create custom delegate
+	delegate := list.NewDefaultDelegate()
+	delegate.Styles.SelectedTitle = delegate.Styles.SelectedTitle.Copy().
```

golangci-lint (1.64.8)

SA1019: delegate.Styles.SelectedTitle.Copy is deprecated: to copy just use assignment (i.e. a := b). All methods also return a new style. 

(staticcheck)